### ISSUE TITLE
[HotspotEditorModal] Tooltip stays open

### ISSUE THREAD

**Issue by bjornalm** [carbon-design-system/carbon-addons-iot-react#2415]
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

The tooltip of a hotspot sometimes stays open or reopens  even when clicking on other elements in the HotspotEditorModal. 

### To Reproduce

1. Go to https://next.carbon-addons-iot-react.com/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-hotspoteditormodal--width-existing-hotspots
2. Click on the My device hotspot.
3. Click on any of the input controls on the right side

### Expected behavior

A click anywhere outside the hotspot tooltip should close the tooltip.

### Environment/versions:

- OS: MacOS
- Browser: chrome

### Additional context
![bug](https://user-images.githubusercontent.com/5167091/122181423-a17b8400-ce89-11eb-8b29-3f67e29b24bb.gif)

### Specific timeline issues / requests

Do you want this work within a specific time period? Is it related to an
upcoming release?

_NB: The core contributors will try to work with your timeline, but it's not
guaranteed. The earlier you make a request in advance of a desired delivery
date, the better!_

**Comment by JordanWSmith15** [comment]
@bjornalm Is this ready to close? Seems the fix is merged

**Comment by bjornalm** [comment]
> @bjornalm Is this ready to close? Seems the fix is merged

Nope, there is no PR for this issue. It was only mentioned in another PR that was merged.
